crates/sui-light-client/src/checkpoint.rs

```diff
@@ -155,20 +142,10 @@ fn merge_checkpoint_lists(list1: &CheckpointsList, list2: &CheckpointsList) -> V
 
 /// Downloads the list of end of epoch checkpoints from the archive store
```

Comment:
can we rename the function to `sync_checkpoint_list_to_latest_using_checkpoint_bucket` or something similar?